src/lib/pages/collection-list/components/Collections.tsx

```diff
@@ -0,0 +1,69 @@
+import { Box } from "@chakra-ui/react";
+import type { ChangeEvent } from "react";
+import { useState } from "react";
+
+import { TextInput } from "lib/components/forms";
+import { Pagination } from "lib/components/pagination";
+import { usePaginator } from "lib/components/pagination/usePaginator";
+import { useCollectionsPagination } from "lib/services/collectionService";
+
+import CollectionList from "./CollectionList";
+
+const Collections = () => {
+  const {
+    pagesQuantity,
+    currentPage,
+    setCurrentPage,
+    pageSize,
+    setPageSize,
+    offset,
+  } = usePaginator({
+    initialState: {
+      pageSize: 10,
+      currentPage: 1,
+      isDisabled: false,
+    },
+  });
+
+  const [searchKeyword, setSearchKeyword] = useState("");
+
+  const { data: collections, isLoading } = useCollectionsPagination(
+    pageSize,
+    offset,
+    searchKeyword
+  );
+
+  const onPageSizeChange = (e: ChangeEvent<HTMLSelectElement>) => {
+    const size = Number(e.target.value);
+    setPageSize(size);
+    setCurrentPage(1);
+  };
+
+  return (
+    <Box>
+      <TextInput
+        variant="fixed-floating"
+        value={searchKeyword}
+        setInputState={setSearchKeyword}
+        placeholder="Search by collection name"
+        size="md"
+        mb="32px"
+      />
```

Comment:
Please change to

```jsx
<InputWithIcon
placeholder="Search with Collection Name"
value={searchKeyword}
autoFocus
onChange={(e) => setSearchKeyword(e.target.value)}
size={{ base: "md", md: "lg" }}
/>
```

Reply:
please add margin top to collection list @James0126

Reply:
okay